<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTAFailProfileNoiseWgt</class>
 <widget class="QWidget" name="GTAFailProfileNoiseWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Noise profile</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>mean</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="meanLE"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>sigma</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sigmaLE"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>seed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="seedLE"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
